KelynPNjeri/Store-Manager
# #161435377 Creating Database.

#### What does this PR do?
It contains the database configurations need all through the API.

#### Description of Task to be completed?
The tasks to be completed were:

- Creation of a database.

- Creation of the Database Tables

#### How should this be manually tested?
Clone the repository using `git clone https://github.com/KelynPNjeri/Store-Manager.git`.
Using the terminal change directory into where you cloned the repository.
Create a virtual environment and activate it.
Install project requirements into your virtual environment from `requirements.txt` using `pip install -r requirements.txt`.
Create a database in Postgres named store-manager 
Open the project with your favorite code editor.
Ensure you've got python installed
Navigate to the database.py file.
Change the variables in connect with the ones relevant to you.
Run the file using `python database.py`.

#### Any background context you want to provide?
None.

#### What are the relevant pivotal tracker stories?
[#161435377](https://www.pivotaltracker.com/story/show/161435377)

#### Screenshots (if appropriate)
None.
